data/locale/zh-CN.ini

```diff
@@ -1,8 +1,7 @@
-﻿NDIPlugin.Default="默认"
+NDIPlugin.Default="默认"
```

Comment:
Weird change; please revert this one line

Reply:
I can't fix that because I'm unable to find out what changed. I tried again but Git doesn't show this in the diff but still commits it. For me it seems Git fixes something on that line? idk

Reply:
Okay fixed it somehow, though there seems to be something wrong about this line...